timon/plugins/http_server/views.py

```diff
@@ -95,11 +90,11 @@ async def search_probe_in_queue(probename):
     """
     Search a probe in the queue, and returns it if it exists else returns a 404
     """
-    heap = app.tmoncfg.get_queue().heap
-    for item in heap:
-        if item[1] == probename:
-            return jsonify(item)
-    return f"probe {probename} not found in heap", 404
+    queue = app.tmoncfg.get_queue()
+    prb_info = queue.get_probe_info_in_heap()
```

Comment:
how does this work.
wher is probename used?

Reply:
ohhh shit